### PR TITLE
feat(metrics): add system metrics

### PR DESCRIPTION
**Pull request by MaksimChepelev**
System Available Disk
System Available Memory
System Total CPU

Closes: ENG-728, ENG-729, ENG-730

### CODE REVIEW

**Comment by MaksimChepelev** [comment]
Merge only with controller

File: daemon/src/org/eclipse/iofog/resource_consumption_manager/ResourceConsumptionManager.java [hunk 1]
```diff
@@ -166,6 +174,38 @@ private float getCpuUsage() {
...
+		CommandShellResultSet<List<String>, List<String>> resultSet = executeCommand(MEM_AVAILABLE);
+		long memInKB = Long.parseLong(parseOneLineResult(resultSet));
+		return memInKB * 1024;
+	}
+
+	private float getTotalCpu() {
+        if (SystemUtils.IS_OS_WINDOWS) {
+            return 0;
+        }
+		final String CPU_USAGE = "grep 'cpu' /proc/stat | awk '{usage=($2+$3+$4)*100/($2+$3+$4+$5+$6+$7+$8+$9)} END {print usage}'";
```
Comment (MaksimChepelev) [inline daemon/src/org/eclipse/iofog/resource_consumption_manager/ResourceConsumptionManager.java]: add here comment with link: https://github.com/Leo-G/DevopsWiki/wiki/How-Linux-CPU-Usage-Time-and-Percentage-is-calculated

File: daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java [hunk 2]
```diff
@@ -77,20 +77,29 @@ public static String getStatusReport() {
...
+		float totalCpu = resourceConsumptionManagerStatus.getTotalCpu();
+
 		String connectionStatus = fieldAgentStatus.getControllerStatus() == ControllerStatus.OK ? "ok" :
 				(fieldAgentStatus.getControllerStatus() == ControllerStatus.BROKEN_CERTIFICATE ? "broken" : "not provisioned");
 		result.append("ioFog daemon                : ").append(supervisorStatus.getDaemonStatus().name());
-		result.append("\\nMemory Usage                : about ").append(String.format("%.2f", resourceConsumptionManagerStatus.getMemoryUsage())).append(" MiB");
+		result.append("\\nMemory Usage                : about ").append(String.format("%.2f MB", resourceConsumptionManagerStatus.getMemoryUsage()));
 		if (diskUsage < 1)
-			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f", diskUsage * 1024)).append(" MiB");
+			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f MB", diskUsage * 1024));
```
Comment (Railag) [inline daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java]: MiB / MB

File: daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java [hunk 3]
```diff
@@ -77,20 +77,29 @@ public static String getStatusReport() {
...
 		float diskUsage = resourceConsumptionManagerStatus.getDiskUsage();
+		double availableDisk = resourceConsumptionManagerStatus.getAvailableDisk() / 1024. / 1024.;
+		double availableMemory = resourceConsumptionManagerStatus.getAvailableMemory() / 1024. / 1024.;
+		float totalCpu = resourceConsumptionManagerStatus.getTotalCpu();
+
 		String connectionStatus = fieldAgentStatus.getControllerStatus() == ControllerStatus.OK ? "ok" :
 				(fieldAgentStatus.getControllerStatus() == ControllerStatus.BROKEN_CERTIFICATE ? "broken" : "not provisioned");
 		result.append("ioFog daemon                : ").append(supervisorStatus.getDaemonStatus().name());
-		result.append("\\nMemory Usage                : about ").append(String.format("%.2f", resourceConsumptionManagerStatus.getMemoryUsage())).append(" MiB");
+		result.append("\\nMemory Usage                : about ").append(String.format("%.2f MB", resourceConsumptionManagerStatus.getMemoryUsage()));
```
Comment (Railag) [inline daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java]: MiB / MB

File: daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java [hunk 4]
```diff
@@ -77,20 +77,29 @@ public static String getStatusReport() {
...
 		if (diskUsage < 1)
-			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f", diskUsage * 1024)).append(" MiB");
+			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f MB", diskUsage * 1024));
 		else
-			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f", diskUsage)).append(" GiB");
-		result.append("\\nCPU Usage                   : about ").append(String.format("%.2f", resourceConsumptionManagerStatus.getCpuUsage())).append("%");
+			result.append("\\nDisk Usage                  : about ").append(String.format("%.2f GB", diskUsage));
```
Comment (Railag) [inline daemon/src/org/eclipse/iofog/status_reporter/StatusReporter.java]: GiB/GB

**Pull request merged**